course/src/main/java/org/openedx/course/presentation/ui/CourseUI.kt

```diff
@@ -915,6 +966,64 @@ fun SubSectionUnitsList(
     }
 }
 
+@Composable
+fun CourseDatesBanner(
+    modifier: Modifier,
+    banner: CourseDatesBannerInfo,
+    resetDates: () -> Unit,
+) {
+    val cardModifier = modifier
+        .background(
+            MaterialTheme.appColors.cardViewBackground,
+            MaterialTheme.appShapes.datesBannerShape
```

Comment:
No need to define a custom shape for this, we can use `MaterialTheme.appShapes.material.medium` for this